docs/docsite/rst/user_guide/playbooks_vault.rst

```diff
@@ -37,9 +44,46 @@ If you are using a script instead of a flat file, ensure that it is marked as ex
 
 This is something you may wish to do if using Ansible from a continuous integration system like Jenkins.
 
+The :option:`--vault-id <ansible-pull --vault-id>` option can also be used with the :ref:`ansible-pull` command if you wish, though this would require distributing the keys to your nodes, so understand the implications -- vault is more intended for push mode.
+
+*Prior to Ansible 2.4*
+
```

Comment:
the same here, this is in 'addition' as the old 'single vault options' still work